docs/reference/config.rst

```diff
@@ -1,39 +1,37 @@
 Configuration
 =============
 
-Beets has an extensive configuration system that lets you customize nearly
-every aspect of its operation. To configure beets, you create a file called
-``config.yaml``. The location of the file depend on your platform (type ``beet
-config -p`` to see the path on your system):
-
-* On Unix-like OSes, write ``~/.config/beets/config.yaml``.
-* On Windows, use ``%APPDATA%\beets\config.yaml``. This is usually in a
-  directory like ``C:\Users\You\AppData\Roaming``.
-* On OS X, you can use either the Unix location or ``~/Library/Application
+Beets has an extensive configuration system which lets you customize nearly
+every aspect of beets. To configure beets, a configuration file called
+``config.yaml`` must be created. The location of ``config.yaml`` depends on your operating system (OS).
```

Comment:
I actually kind of like mentioning `config -p` first because it tells the reader that they can skip the bullet points if they want. If they just care about where the file is on their system, they can just type the command—in fact, they can remember this for the future in case things change or they use a custom location. The bullet points are just for full details and most readers can just skip right over them.

I also marginally prefer “platform” to “operating system (OS).” It reads a little more naturally, I think, and avoids a jargony feeling.

Reply:
Ok I'll undo the "platform" change and move `config -p` back to its original location

Reply:
I see. I'll undo the "platform" change and move config -p back to its original location.